lib/plugins/aws/invokeLocal/index.js

```diff
@@ -140,7 +140,7 @@ class AwsInvokeLocal {
   getConfiguredEnvVars() {
     const providerEnvVars = this.serverless.service.provider.environment || {};
     const functionEnvVars = this.options.functionObj.environment || {};
-    return _.merge(providerEnvVars, functionEnvVars);
+    return _.pickBy(_.merge(providerEnvVars, functionEnvVars), (v) => v !== null);
```

Comment:
Let's try to not use lodash-based methods if possible - we're trying to limit their use as much as possible and eventually drop that dependency

Reply:
I did it, but it doesn't look clean to me. Is that OK with you?

Reply:
Let's keep it this way and after dropping support for Node 10.x we can refactor it with `entries/fromEntries` notation